LinqToRest/Async/Infrastructure/QueryableMethods.cs

```diff
@@ -0,0 +1,680 @@
+﻿// from https://raw.githubusercontent.com/dotnet/efcore/master/src/EFCore/Query/QueryableMethods.cs
+
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using JetBrains.Annotations;
+
+namespace Messerli.LinqToRest.Async.Infrastructure
+{
+    /// <summary>
+    ///     A class that provides reflection metadata for translatable LINQ methods.
+    /// </summary>
+    public static class QueryableMethods
```

Comment:
Could this be internal?
```suggestion
    internal static class QueryableMethods
```

Reply:
Fixed in af513b81e206a3058dd49bfb5fa2723dec122588